static/app/views/dashboards/datasetConfig/spans.tsx

```diff
@@ -54,8 +52,8 @@ const EAP_AGGREGATIONS = ALLOWED_EXPLORE_VISUALIZE_AGGREGATES.reduce((acc, aggre
     parameters: [
       {
         kind: 'column',
-        columnTypes: ['string', 'integer', 'number', 'duration', 'date', 'boolean'],
-        defaultValue: 'user',
+        columnTypes: ['number'],
```

Comment:
Since I changed this to only `'number'`, we can't simply render the full key anymore before all of the metadata loads up, the field has a moment where it flickers from undefined to the value and we get complaints about a field going from uncontrolled to controlled.